lib/cli.js

```diff
@@ -534,10 +534,10 @@ module.exports = (argv) => {
  * @returns {void}
  */
 function handleError(err) {
-	console.log(err.stack); // eslint-disable-line no-console
+	process.stderr.write(err.stack);
 	const exitCode = typeof err.code === 'number' ? err.code : 1;
 
-	process.exit(exitCode); // eslint-disable-line no-process-exit
+	process.exitCode = exitCode;
```

Comment:
Set an exitCode and wait for node to exit when any pending work is complete. This means we can remove another `// eslint-disable-line` comment. The node docs [recommend against using `process.exit()`](https://nodejs.org/api/process.html#process_process_exit_code) where possible.